### ISSUE TITLE
incorrect counting of the number of "total" operations

### ISSUE THREAD

**Issue by MykhailoKhil** [hoodoo-digital/locust-influxdb-listener#5]
Hello there! 
I faced with problem when count total operation. In Grafana when I took request 
```
SELECT count("counter")
FROM "locust_requests"
WHERE $timeFilter
```

actual number 43310 
![image](https://user-images.githubusercontent.com/34770700/111123909-8fcbcd00-8578-11eb-8724-156bb1fa87b2.png)

but expected 73365 by locust stats 
![image](https://user-images.githubusercontent.com/34770700/111124025-ad993200-8578-11eb-9036-2c7d492a6456.png)

**Comment by MykhailoKhil** [comment]
Here my fix. Sorry cant do pull request for now. I use 'counter': self.env.stats.num_requests for __listen_for_requests_events()
[__init__.py.zip](https://github.com/hoodoo-digital/locust-influxdb-listener/files/6140518/__init__.py.zip)

**Comment by pjcalvo** [comment]
Closed on PR#6 https://github.com/hoodoo-digital/locust-influxdb-listener/pull/8
